### PR TITLE
chore : Update localization values

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes :

I worked on updating the localization values for 
- labels
- message
- server

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Review by chirag-madlani — APPROVED** [review]

**Review by chirag-madlani — DISMISSED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 1]
```diff
@@ -489,92 +490,92 @@
...
-    "field-region-name-description": "Region name. Required when using AWS Credentials.",
+    "field-region-name-description": "Region name is required when using AWS Credentials.",
     "field-text-is-invalid": "{{fieldText}} is invalid.",
     "field-text-is-required": "{{fieldText}} is required.",
     "field-timeout-description": "Connection Timeout",
-    "field-use-aws-credentials-description": "Indicates whether to use aws credentials when connecting to OpenSearch in AWS.",
-    "field-use-ssl-description": "Indicates whether to use SSL when connecting to ElasticSearch. By default, we will ignore SSL settings.",
-    "field-verify-certs-description": "Indicates whether to verify certificates when using SSL connection to ElasticSearch. Ignored by default. Is set to true, make sure to send the certificates in the property `CA Certificates`.",
+    "field-use-aws-credentials-description": "Indicates whether to use AWS credentials when connecting to OpenSearch in AWS.",
+    "field-use-ssl-description": "This indicates whether to use SSL when connecting to ElasticSearch. By default, we will ignore SSL settings.",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:569]: Elasticsearch

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 2]
```diff
@@ -585,42 +586,42 @@
...
-    "no-selected-dbt": "No source selected for DBT Configuration.",
+    "no-selected-dbt": "No source selected for dbt Configuration.",
     "no-service-connection-details-message": "{{serviceName}} doesn't have connection details filled in. Please add the details before scheduling an ingestion job.",
     "no-team-found": "No team found.",
     "no-terms-found": "No terms found",
     "no-terms-found-for-search-text": "No terms found for {{searchText}}",
     "no-user-available": "No user available",
     "no-users": "There are no users {{text}}",
-    "om-description": "centralized metadata store, discover, collaborate and get your data right",
+    "om-description": "Centralized metadata store, to Discover, collaborate and get your data right.",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:621]: discover

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 3]
```diff
@@ -632,19 +633,19 @@
...
     "select-interval-type": "Select interval type",
     "select-interval-unit": "Select interval unit",
     "select-team": "Please select a team type",
     "select-token-expiration": "Select Token Expiration",
-    "special-character-not-allowed": "Special characters are not allowed",
-    "sql-query-tooltip": "Queries returning 1 or more rows will result in the test failing.",
+    "special-character-not-allowed": "Special characters are not allowed.",
+    "sql-query-tooltip": "Queries returning one or more rows will result in the test failing.",
     "successfully-completed-the-tour": "You’ve successfully completed the tour.",
     "sure-to-remove": "Are you sure you want to remove the",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:646]: Will some code be added here? This line is incomplete

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 4]
```diff
@@ -632,19 +633,19 @@
     "restore-entities-error": "Error while restoring {{entity}}",
     "restore-entities-success": "{{entity}} restored successfully",
     "scopes-comma-separated": "Scopes value comma separated",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:635]: Add the Scopes value, separated by commas.

I hope that's the same meaning being conveyed.

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 5]
```diff
@@ -585,42 +586,42 @@
...
     "no-users": "There are no users {{text}}",
-    "om-description": "centralized metadata store, discover, collaborate and get your data right",
+    "om-description": "Centralized metadata store, to Discover, collaborate and get your data right.",
     "permanently-delete-metadata": "Permanently deleting this {{entityName}} will remove its metadata from OpenMetadata permanently.",
     "permanently-delete-metadata-and-dependents": "Permanently deleting this {{entityName}} will remove its metadata, as well as the metadata of {{dependents}} from OpenMetadata permanently.",
-    "pipeline-trigger-success-message": "Pipeline triggered successfully",
+    "pipeline-trigger-success-message": "Pipeline triggered successfully!",
     "profile-sample-percentage-message": "Set the Profiler value as percentage",
     "profile-sample-row-count-message": " Set the Profiler value as row count",
     "profiler-timeout-seconds-message": "Optional number setting the timeout in seconds for the profiler. If the timeout is reached the profiler will wait for any pending queries to terminated its execution.",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:627]: Unclear message.... Is it supposed to be:

For the Profiler, add the duration in seconds for timeout (optional). The profiler will wait for any pending queries to be executed, or will terminate them once it reaches the timeout.

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 6]
```diff
@@ -585,42 +586,42 @@
...
+    "no-notification-found": "No notifications found",
     "no-permission-for-action": "You do not have the necessary permissions to perform this action.",
     "no-permission-to-view": "You do not have the necessary permissions to view this data.",
     "no-policy-found-for": "No policy found for",
-    "no-profiler-enabled-summary-message": "Profiler is not enabled for the table.",
-    "no-profiler-message": "Data Profiler is an optional configuration in Ingestion. Please enable the data profiler by following the documentation",
+    "no-profiler-enabled-summary-message": "Profiler is not enabled for this table.",
+    "no-profiler-message": "Data Profiler is an optional configuration in Ingestion. Please enable the data profiler by following the documentation.",
     "no-roles-assigned": "No roles assigned",
     "no-roles-found-for": "No roles found for",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:611]: Any personalization code to be added here? Incomplete sentence

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 7]
```diff
@@ -585,42 +586,42 @@
...
+    "no-ingestion-description": "To view Ingestion Data, run the metadata ingestion. Please refer to this doc to schedule the",
     "no-inherited-roles-found": "No inherited roles found",
-    "no-kpi-available-add-new-one": "No KPI's are available, add one by clicking Add KPI button.",
+    "no-kpi-available-add-new-one": "No KPIs are available. Click on the Add KPI button to add one.",
     "no-kpi-found": "No KPI found with name {{name}}",
-    "no-notification-found": "No notifications Found",
+    "no-notification-found": "No notifications found",
     "no-permission-for-action": "You do not have the necessary permissions to perform this action.",
     "no-permission-to-view": "You do not have the necessary permissions to view this data.",
     "no-policy-found-for": "No policy found for",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:607]: any code needed? Incomplete sentence

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 8]
```diff
@@ -585,42 +586,42 @@
...
     "most-viewed-data-assets": "Displays the most viewed data assets.",
     "new-to-the-platform": "New to the platform?",
-    "no-data-available": "No data available",
+    "no-data-available": "No data available.",
     "no-data-available-for-selected-filter": "No data found. Try changing the filters.",
     "no-execution-runs-found": "No execution runs found for the pipeline.",
     "no-features-data-available": "No features data available",
     "no-ingestion-available": "No ingestion data available",
-    "no-ingestion-description": "To view Ingestion Data, run the MetaData Ingestion. Please refer to this doc to schedule the",
+    "no-ingestion-description": "To view Ingestion Data, run the metadata ingestion. Please refer to this doc to schedule the",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:600]: incomplete sentence

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 9]
```diff
@@ -489,92 +490,92 @@
...
     "enter-display-name": "Enter display name",
     "enter-feature-description": "Enter feature description",
     "enter-interval": "Enter interval",
     "enter-test-case-name": "Enter test case name",
     "entity-already-exists": "{{entity}} already exists.",
-    "entity-owned-by-name": "This Entity is Owned by {{entityOwner}}",
+    "entity-owned-by-name": "This entity is owned by {{entityOwner}}",
     "entity-restored-error": "Error while restoring {{entity}}",
     "entity-restored-success": "{{entity}} restored successfully",
     "fetch-dbt-files": "Available sources to fetch dbt catalog and manifest files.",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:560]: Would it be correct to say:

These are the available sources to fetch dbt catalog and manifest files.

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 10]
```diff
@@ -489,92 +490,92 @@
...
     "create-new-glossary-guide": "A Glossary is a controlled vocabulary used to define the concepts and terminology in an organization. Glossaries can be specific to a certain domain (for e.g., Business Glossary, Technical Glossary). In the glossary, the standard terms and concepts can be defined along with the synonyms, and related terms. Control can be established over how and who can add the terms in the glossary.",
-    "create-or-update-email-account-for-bot": "Changing account email will update or create a new bot user",
+    "create-or-update-email-account-for-bot": "Changing the account email will update or create a new bot user.",
     "created-this-task-lowercase": "created this task",
-    "data-asset-has-been-action-type": "Data asset has been {{actionType}}",
+    "data-asset-has-been-action-type": "Data Asset has been {{actionType}}",
     "data-insight-page-views": "Displays the number of times a dataset type was viewed.",
     "data-insight-subtitle": "Get a single pane view of the health of all your data assets over time.",
     "dbt-run-result-http-path-message": "dbt run results http path to extract the test results information.",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:539]: Would it be correct to say

dbt runs the results on an http path to extract the test results.

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json [hunk 11]
```diff
@@ -489,92 +490,92 @@
...
-    "admin-only-action": "Only admin can perform this action.",
+    "admin-only-action": "Only an Admin can perform this action.",
     "advanced-search-message": "Discover the right data assets using the syntax editor with and/or conditions.",
     "alerts-description": "Stay current with timely alerts using webhooks.",
-    "alerts-destination-description": "Send notifications to Slack, MS Teams, Email, or and use Webhooks.",
+    "alerts-destination-description": "Send notifications to Slack, MS Teams, Email, or use Webhooks.",
     "alerts-filter-description": "Specify the change events to narrow the scope of your alerts.",
     "alerts-trigger-description": "Trigger for all data assets or a specific entity.",
     "all-charts-are-mapped": "All charts are mapped with existing KPIs.",
     "and-followed-owned-by-name": "and followed team owned by {{userName}}",
```
Comment (ShilpaVernekar) [inline openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:516]: Would it be correct to say

The team you are following is owned by {{userName}}

**Review by ShilpaVernekar — COMMENTED** [review]
@chirag-madlani @Sachin-chaurasiya  - Re-adding the comments from the doc. Please verify if that would be the correct thing to say

**Comment by Sachin-chaurasiya** [comment]
> @chirag-madlani @Sachin-chaurasiya - Re-adding the comments from the doc. Please verify if that would be the correct thing to say

Thank you @ShilpaVernekar, I will work on the comments.

File: openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json (hunk 8 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json:601]: This is used with other keys to make it a complete sentence.

**Review by chirag-madlani — APPROVED** [review]

**Review by ShilpaVernekar — APPROVED** [review]
Sounds good. Thanks @Sachin-chaurasiya

**Pull request merged**